.eslintignore

```diff
@@ -0,0 +1,4 @@
+**/dist/
+**/node_modules/
+public
+vite.config.ts
```

Comment:
Add `cypress.config.ts` like in `documentation-homepage` repo? And also to avoid a console warning when running the lint.